Navitia.php

```diff
@@ -2,173 +2,78 @@
 
 namespace CanalTP\NavitiaPhp;
 
-use CanalTP\NavitiaPhp\Api\Coords;
-use CanalTP\NavitiaPhp\Api\Lines;
-use Symfony\Component\Yaml\Yaml;
-use Symfony\Component\OptionsResolver\OptionsResolver;
 use GuzzleHttp\Client;
 use GuzzleHttp\Exception\ClientException;
-use GuzzleHttp\Exception\RequestException;
-use GuzzleHttp\Exception\ServerException;
 use CanalTP\NavitiaPhp\Exception\NavitiaException;
-use CanalTP\NavitiaPhp\Api\Coverage;
-use CanalTP\NavitiaPhp\Api\TrafficReports;
 
 class Navitia
 {
-    private $httpClient;
     /**
-     * @var string
+     * @var Client
      */
-    private $defaultCoverage;
```

Comment:
The default coverage is a massively used feature. I had the intention to add the defaultCoverage into required configuration.
Maybe you think another way?